rootfs/api/models/app.py

```diff
@@ -458,7 +452,16 @@ def _default_scale(self, user, release):
         else:
             structure = {'web': 1}
 
-        self.scale(user, structure)
+        if "scale" in dir(self._scheduler):
```

Comment:
Doing this whole shenanigans to keep the mock scheduler happy, for now. Took away some other changes I was doing just to keep it happy.